Assets/MRTK/SDK/Features/UX/Scripts/BoundsControl/BoundsControl.cs

```diff
@@ -784,7 +784,7 @@ private void Update()
                 // also only use proximity effect if nothing is being dragged or grabbed
                 if (!wireframeOnly && currentPointer == null)
                 {
-                    proximityEffect.UpdateScaling(Vector3.Scale(TargetBounds.center, TargetBounds.gameObject.transform.lossyScale) + Target.transform.position, currentBoundsExtents);
+                    proximityEffect.UpdateScaling(TargetBounds.transform.TransformPoint(TargetBounds.center), currentBoundsExtents);
```

Comment:
We no longer user `Target.transform` for anything here. Are there any cases where that might be a problem?

Reply:
Could you elaborate a little bit more? I replaced Target with TargetBounds so there is no problem right?

Reply:
I think I'm not sure the difference between `Target` and `TargetBounds` and wondering if ignoring `Target` now might cause us to ignore some positional offset or something

Reply:
Resolved offline. The proximity effect really only cares about the target's collider (`TargetBounds`), regardless of the GameObject's position itself